src/lib/bootloader/autoyast_converter.rb

```diff
@@ -101,6 +101,13 @@ def import_default(data, default)
           val = data.global.public_send(key)
           next unless val
 
+          # import resume only if device exists (bsc#1187690)
+          resume = val[/(?:\s|\A)resume=(\S+)/, 1]
+          if resume && !Yast::BootStorage.staging.find_by_any_name(resume)
+            log.warn "remove resume parameter for non existing #{resume}"
```

Comment:
I know it's only a warning in the logs, but it could be phrased to explain the problem better